desktop/main-app/scripts/esbuild/plugin/dotEnvFlowPlugin.ts

```diff
@@ -0,0 +1,63 @@
+import { Plugin, PluginBuild } from "esbuild";
+import dotenvFlow, { DotenvConfigOptions } from "dotenv-flow";
+
+function parseWithEnvObject(orgEnv: Record<string, string | undefined>): Record<string, string> {
+    return Reflect.ownKeys(orgEnv)
+        ?.map(item => {
+            if (typeof item === "string") {
+                const tmp: Record<string, any> = {};
+                tmp[`process.env.${item}`] = `"${orgEnv?.[item]}"`;
+                return tmp;
+            }
+            return {};
```

Comment:
This will case error by esbuild replace. We must replace by string not an variable
```
  const result: Record<string, any> = {};
            if (typeof item === "string" && orgEnv?.[item]) {
                result[`process.env.${item}`] = `"${orgEnv?.[item]}"`;
            }
            return result;
``

Reply:
```js
result[`process.env.${item}`] = JSON.stringify(orgEnv?.[item]);
```